webapp/src/applications/Organization/Gestion/Jobs/Create/CreatingCard.tsx

```diff
@@ -98,26 +125,29 @@ function CreatingCard() {
                             </DateFnsProvider>
                         </Grid>
                         <Grid item xs={12} sm={6}>
-                            <TextField
+                            <Select
+                                value={level}
                                 fullWidth
                                 required
-                                inputProps={{ min: 0 }}
-                                label="Tokens gagnés"
-                                type='number'
-                                id="outlined-start-adornment"
-                                InputProps={{
-                                    endAdornment: <InputAdornment position="end">tokens</InputAdornment>,
-                                }}
-                            />
+                                onChange={(e) => setLevel(e.target.value)}
```

Comment:
:)

```suggestion
                                onChange={(e: React.FormEvent<HTMLInputElement>) => setLevel(e.target.value)}
```

Reply:
`Type '(e: FormEvent<HTMLInputElement>) => void' is not assignable to type '(event: SelectChangeEvent<Object>, child: ReactNode) => void'.
  Types of parameters 'e' and 'event' are incompatible.
    Type 'SelectChangeEvent<Object>' is not assignable to type 'FormEvent<HTMLInputElement>'.
      Type 'Event & { target: { value: Object; name: string; }; }' is missing the following properties from type 'FormEvent<HTMLInputElement>': nativeEvent, isDefaultPrevented, isPropagationStopped, persistts(2322)
Select.d.ts(96, 3): The expected type comes from property 'onChange' which is declared here on type 'IntrinsicAttributes & SelectProps<Object>'`